test/onigumo_test.exs

```diff
@@ -44,16 +43,18 @@ defmodule OnigumoTest do
   test("download URLs from the input file", %{tmp_dir: tmp_dir}) do
     expect(HTTPoisonMock, :get!, length(@urls), &get!/1)
 
+    env_path = Application.get_env(:onigumo, :input_path)
+    input_path = Path.join(tmp_dir, env_path)
```

Comment:
```suggestion
tmp_path = Application.get_env(:onigumo, :input_path) |> Path.join(tmp_dir, &1)
```

Unfortunately it is not working

Reply:
How come the tests pass then?

Reply:
Oh. I see. You mean that the suggested more concise approach with `&1` does not work. That’s unfornutately true. `&` captures can be only used in anonymous functions, not to override argument order when piping. Nice try though. 😃